### ISSUE TITLE
hitSlop is not filtered out when Platform.OS is web

### ISSUE THREAD

**Issue by kpcurai** [tamagui/tamagui#2150]
### Current Behavior

Tamagui doesn't filter out the RN-only prop "hitSlop" on web, resulting in the console error:
```
Warning: React does not recognize the `hitSlop` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `hitslop` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
``` 

I've confirmed this on both `Button` and `RadioGroup.Item`

### Expected Behavior

Tamagui only applies a provided `hitSlop` prop on native devices.

### Tamagui Version

```markdown
1.84.2
```

### Platform (Web, iOS, Android)

```markdown
Web
```

### Reproduction

```markdown
https://github.com/kpcurai/super-duper-octo-engine/commit/2b6bfdf04398d8e247d2bfc2b4f2163fad431b9e
```

### System Info

```markdown
System:
    OS: macOS 13.6.1
    CPU: (8) arm64 Apple M1 Pro
    Memory: 106.11 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.6.0 - /opt/homebrew/bin/node
    Yarn: 4.0.2 - /opt/homebrew/bin/yarn
    npm: 10.2.4 - /opt/homebrew/bin/npm
    Watchman: 2024.01.15.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 120.0.6099.234
    Safari: 17.1
  npmPackages:
    @babel/runtime: ^7.18.9 => 7.21.0 
    @manypkg/cli: ^0.19.1 => 0.19.2 
    check-dependency-version-consistency: ^3.0.3 => 3.0.3 
    eslint: ^8.21.0 => 8.39.0 
    node-gyp: ^9.3.1 => 9.3.1 
    prettier: ^2.7.1 => 2.8.8 
    turbo: ^1.10.3 => 1.10.3 
    typescript: ^5.3.3 => 5.3.3
```

**Comment by natew** [comment]
Fixed in recent versions.

**Comment by kpcurai** [comment]
I observe this still in v1.88.13

**Comment by natew** [comment]
Should be fixed in 1.88.17 lmk
